### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by ixayldz** [Thiry1/CSGO_Match_Finder#1]
is it working still ?

**Comment by ixayldz** [comment]
http://pastebin.com/n20AiiRf

**Comment by qwexvf** [comment]
as i heard he is a little busy with school stuff. btw I'm having the same thing here.

**Comment by ixayldz** [comment]
i try to find people for fix but there is no scala coder :S

**Comment by ixayldz** [comment]
http://steamcommunity.com/id/jesustar/ can u add me

**Comment by qwexvf** [comment]
Yeah, thats the problem which is too hard to use a framework that you don't know.
for me i mostly used Python and php frameworks which is pretty cheap.

if you trying to learn those things we can help each other.

**Comment by ixayldz** [comment]
https://imgur.com/2GPDD1U
